crates/wac-resolver/src/lib.rs

```diff
@@ -149,3 +145,64 @@ pub fn packages<'a>(
     visitor.visit(document)?;
     Ok(keys)
 }
+
+#[derive(Debug, Error)]
+/// Error for WAC commands
+pub enum CommandError {
+    /// General errors
+    #[error("Error: `{0}`")]
```

Comment:
I think the errors typically start with lowercase letters. Also it seems like we're not really adding much context to the errors, so perhaps a `#[transparent]` marker would be better. Those would also automatically add the `From` impls for those types.